docs/roadmap.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap
+
+## Version 1.0.0
+
+There are a few PRs against Webpack to use directly or indirectly this project.
```

Comment:
Can you link to these PRs? it would be good to know how this project is potentially being used.